training/tf/chunkparser.py

```diff
@@ -418,9 +420,9 @@ def test_parsing(self):
         items = []
         for p in range(16):
             # generate first 360 bits
-            h = np.packbits([int(x) for x in planes[p][0:360]]).tobytes().hex()
+            h = np.packbits([int(x) for x in planes[p][0:(NUM_INTERSECTIONS-1)]]).tobytes().hex()
             # then add the stray single bit
-            h += str(planes[p][360]) + "\n"
+            h += str(planes[p][NUM_INTERSECTIONS-1]) + "\n"
             items.append(h)
         # then side to move
         items.append(str(int(planes[17][0])) + "\n")
```

Comment:
I'm guessing this shouldn't still include `int`

Reply:
It doesn't matter since the stm generated for this test are 0 or 1 (https://github.com/Hersmunch/leela-zero/blob/937fecc88ccd46354f5ff297a67a44740f6e8fab/training/tf/chunkparser.py#L398), but it's appropriate to remove `int`, yes. Then we may add test cases with fractional stm.
I'm not exactly sure that this test will pass by the way; I don't know whether running parse.py to do training invokes this test somewhere.